packages/address-book-controller/package.json

```diff
@@ -23,6 +23,8 @@
   "scripts": {
     "build:docs": "typedoc",
     "changelog:validate": "../../scripts/validate-changelog.sh @metamask/address-book-controller",
+    "publish-preview": "yarn npm publish --tag preview",
```

Comment:
What do you think about adding existence checks for these scripts to `constraints.pro`? I've added ones for the other scripts so you should be able to copy and paste.

Reply:
Good idea!

Reply:
Done in https://github.com/MetaMask/controllers/pull/965/commits/1ecd8aa12b30cfc68bfb483384b8ac416440bee6